xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -988,16 +1080,10 @@ struct AppModel: ModelProtocol {
         state: AppModel,
         environment: AppEnvironment
     ) -> Update<AppModel> {
-        guard let gatewayURL = environment.data.noosphere.gatewayURL else {
-            logger.log("No gateway configured. Skipping sync.")
-            return Update(state: state)
-        }
-        
         var model = state
         model.lastGatewaySyncStatus = .pending
-        
-        logger.log("Syncing with gateway: \(gatewayURL.absoluteString)")
-        let fx: Fx<AppAction> = environment.data.syncSphereWithGateway()
+        logger.log("Syncing with gateway: \(model.gatewayURL)")
```

Comment:
We no longer have synchronous access to this property on the actor, so we use the copy on the model instead (the update fn keeps it up-to-date)